applications/nrf_desktop/README.rst

```diff
@@ -1860,14 +1860,26 @@ Bootloader
 
 The nRF Desktop application can use one of the following bootloaders:
 
+.. note::
+   See the chosen bootloader subsection below to learn which SoC Series and configurations it supports.
```

Comment:
I believe this note is not required as its clear to user from the below bullet point about the Series and configurations that the bootloader supports.